prime-angular-frontend/src/app/modules/health-auth-site-reg/health-auth-site-reg.routes.ts

```diff
@@ -21,6 +21,13 @@ export class HealthAuthSiteRegRoutes {
   public static ADMINISTRATOR = 'administrator';
   public static SITE_OVERVIEW = 'site-overview';
 
+  public static STEP_CARE_TYPE = 'Care Type';
+  public static STEP_SITE_INFORMATION = 'Site Details';
+  public static STEP_HOUR_OPERATION = 'Hours';
```

Comment:
Can this be `STEP_HOURS_OPERATION` like `HOURS_OPERATION` a few lines earlier

Reply:
it is fixed.